src/redturtle/volto/monkey.py

```diff
@@ -193,3 +194,24 @@ def search_for_similar(*args, **kwargs):
         return original_obj._old_search_for_similar()
 
     return []
+
+
+def plone_restapi_slateblocklinksretriever_call(self, block):
```

Comment:
can you make a pr on plone.restapi too, so we can discard this patch soon?

Reply:
yes